src/browser/browser.ts

```diff
@@ -176,25 +176,29 @@ export class Browser {
   }
 
   async scrollToLoadAllPanels(page: puppeteer.Page, options: ImageRenderOptions): Promise<DashboardScrollingResult> {
-    const scrollDivSelector = '#page-scrollbar,[class*="scrollbar-view"]';
+    const scrollElementSelector = await page.evaluate(() => {
+      const pageScrollbarSelector = '#page-scrollbar';
```

Comment:
Why the `[class*="scrollbar-view"]` was removed? Will this change support older versions of Grafana like 9 and 10?

Reply:
yeah so this is tricky. we need a way to find if the **page** has a scrollbar container. i'm trying to avoid using the feature toggle, because when we then enable that feature toggle by default and remove it we'll run into a problem where we need to update this code as well. but doing `[class*="scrollbar-view"]` is just so generic it won't work. it picks up scrollbar containers in panels, graph legends, etc. the `#page-scrollbar` id was added in grafana 11.1... i can make the `scrollbar-view` selector more specific (e.g. something like `#page-content > div > div > [class="scrollbar-view"]`), but then we'd have to add another different selector for scenes dashboards since it's slightly different and as you can see these selectors would be pretty brittle 😬 any suggestions? 🤔

Reply:
It is tricky 😞. I remember we made a change [here](https://github.com/grafana/grafana-image-renderer/pull/524) after Grafana 11 (probably because of the new #page-scrollbar) and just want to confirm this will not introduce a regression in an older Grafana version.
We need to support retro compatibility with the latest 3 majors and all feature toggles, with and without Scenes.

Reply:
@evictorero ok i've added some horrible code to add multiple more specific fallback selectors

i'm pretty sure this will now be fully backwards/forwards compatible, it's just... horrible 😂 but i can't really think of a better way of doing it.